src/main/java/uk/gov/hmcts/reform/civil/service/PaymentsService.java

```diff
@@ -21,7 +22,12 @@
     private final PaymentsConfiguration paymentsConfiguration;
     private final OrganisationService organisationService;
 
-    public PaymentDto createCreditAccountPayment(CaseData caseData, String authToken) {
+    public PaymentDto createCreditAccountPayment(CaseData caseData, String authToken) throws FeignException {
+        // To be removed, only used for simulating 422 error for testing locally
```

Comment:
is this meant to go in?

Reply:
it is for Raja or Doug to test with their locals, will be removed before merge!